docs/mkdocs/docs/api/basic_json/operator_spaceship.md

```diff
@@ -0,0 +1,70 @@
+# <small>nlohmann::basic_json::</small>operator<=>
+
+```cpp
+// since C++20
+class basic_json {
+    std::partial_ordering operator<=>(const_reference rhs) const noexcept;  // (1)
+
+    template<typename ScalarType>
+    std::partial_ordering operator<=>(const ScalarType rhs) const noexcept; // (2)
+};
+```
+
+1. 3-way compares two JSON values producing a result of type `std::partial_ordering` according to the following rules:
+    - Two JSON values compare with a result of `std::partial_ordering::unordered` if either value is discarded.
+    - If both JSON values are of the same type, the result is produced by 3-way comparing their stored values using their
+      respective `operator<=>`.
+    - Integer and floating-point numbers are converted to their common type and then 3-way compared using their respective
```

Comment:
Would this (and other related docs) be better as `integer numbers are converted to the floating point number type when compared with a floating point number` or is this also covering `signed integer` to `unsigned integer` comparisons?

Reply:
It's covering signed and unsigned integers, yes.

Reply:
Thanks.